src/i18n/locales/de-DE.json

```diff
@@ -252,6 +252,17 @@
   "cmps.designer.SyncButton.syncSuccess": "Der Designer wurde mit den Lightning Knoten synchronisiert",
   "cmps.designer.SyncButton.syncError": "Netzwerk konnte nicht synchronisiert werden",
   "cmps.designer.SyncButton.syncBtnTip": "Graph von Knoten aktualisieren",
+  "cmps.designer.AutoMineButton.autoMineBtnTip": "Neue Blöcke automatisch abbauen",
```

Comment:
Thank you for going through the trouble of looking up these translations but you do not need to modify the non-english locale files yourself. These are managed by the integration with CrowdIn.com. Once the new english strings are merged into `master`, Crowdin will detect them and provides a UI to machine generate and collaborate on the translations into other languages.

You can leave them here since you've already done the work. Just wanted to mention that for future reference.

Reply:
Thanks! made a note to self for the future :-)